### ISSUE TITLE
ClasspathHelper.forWebInfClasses can fail with an NPE

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#90]
```
servletContext.getRealPath() returns a null (as permitted by the spec) on 
WebLogic 10.3.5 with WAR packaging.

servletContext..getResource("/WEB-INF/classes") seems to work in this instance.

Unfortunately, the latter fails on JBoss EAP 5.1.

Patch against trunk attached
```

Original issue reported on code.google.com by `steve....@gmail.com` on 24 Oct 2011 at 11:47

Attachments:
- [forWebInfClasses.txt](https://storage.googleapis.com/google-code-attachments/reflections/issue-90/comment-0/forWebInfClasses.txt)

**Comment by GoogleCodeExporter** [comment]
```
thanks
```

Original comment by `ronm...@gmail.com` on 16 Nov 2011 at 3:04

**Comment by GoogleCodeExporter** [comment]
```
thanks for the patch
fixed on 0.9.7-RC1 and trunk. comments?
```

Original comment by `ronm...@gmail.com` on 2 May 2012 at 3:54
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed 0.9.7
```

Original comment by `ronm...@gmail.com` on 22 May 2012 at 7:47
- Changed state: **Fixed**
